src/features/gameboard/sound.ts

```diff
@@ -1,5 +1,7 @@
-import * as fx from 'wafxr'
+import Generator from 'wasgen'
+
+const gen = new Generator()
 
 export function playTone(frequency: number) {
-  return fx.play({ volume: 0.25, sustain: 0.0237, release: 0.0114, frequency, source: 'sine', harmonics: 1 })
+  gen.play([{ type: 'sine', gain: { a: 0.1, h: 0, d: 0.1, s: 0.02, r: 0.011 } }], frequency, 0.25, gen.now(), gen.now() + 0.2)
```

Comment:
This sound now sounds different than the original sound,
which is probably undesirable.
We can try out different sounds here: https://fenomas.github.io/wafxr/